apps/api/src/app/integrations/usecases/create-integration/create-integration.usecase.ts

```diff
@@ -120,6 +136,16 @@ export class CreateIntegration {
         });
       }
 
+      if (integrationEntity.active) {
```

Comment:
it is running when `isMultiProviderConfigurationEnabled` is `true` because there is nothing that returns or throws between this and the creation of integration. If you do not see something I miss. this is also in the update here: 
https://github.com/novuhq/novu/pull/3898/files#diff-12b817183e64eb3c166b832f4b66f95bc6e89c25d6ef9ab6cdc6e56d2b0ea929R145

Reply:
sorry, what I meant that we shouldn't run this code when the flag `isMultiProviderConfigurationEnabled: false`, like adding to the if statement `if (isMultiProviderConfigurationEnabled && integration.active) {`.
Also I just wanted to ask you to cover this new part of functionality with the e2e test ;)

Reply:
Yeah like I say here https://github.com/novuhq/novu/pull/3898#discussion_r1281491389 there is other parts that is not covered by the feature flag anymore with the assumption of that the flag will not be covered by. 

Regarding the e2e test see this comment here https://github.com/novuhq/novu/pull/3898#discussion_r1281491389